src/Util/Inflector.php

```diff
@@ -42,14 +43,30 @@ private static function getInstance(): InflectorObject
      */
     public static function tableize(string $word): string
     {
-        return class_exists(InflectorFactory::class) ? self::getInstance()->tableize($word) : LegacyInflector::tableize($word);
```

Comment:
This looks like a PHPStan regression to me. What about just adding `// @phpstan-ignore-line`?

Reply:
I don't see it as a regression. As they say in the documentation: 
> PHPStan needs to be able to locate symbols (classes, functions, constants) used in the analyzed codebase.

So it does its job: the class does not exist, it reports the problem. Why not reverse the condition instead?
```php
return class_exists(LegacyInflector::class) ? LegacyInflector::tableize($word) : self::getInstance()->tableize($word);
```

Reply:
Yes reversing the condition is a better idea.